pay-api/src/pay_api/services/paybc_service.py

```diff
@@ -79,13 +78,10 @@ def create_invoice(self, payment_account: PaymentAccount,  # pylint: disable=too
             invoice_number = invoice.id
 
         invoice_url = current_app.config.get('CFS_BASE_URL') + '/cfs/parties/{}/accs/{}/sites/{}/invs/' \
-            .format(payment_account.paybc_party, payment_account.paybc_account, payment_account.paybc_site)
+            .format(payment_account.cfs_party, payment_account.cfs_account, payment_account.cfs_site)
```

Comment:
payment_Account will still have cfs_party  rite? since we moved them to child table

Reply:
The service object will still have it, as it is populated from payment_account.py service class